src/crl/interactivesessions/ShellSubprocess.py

```diff
@@ -110,7 +110,7 @@ def get_start_trigger(self):
 
     @staticmethod
     def _serialize(python_object):
-        return base64.b64encode(pickle.dumps(python_object, protocol=0))
+        return base64.b64encode(pickle.dumps(python_object, protocol=2))
```

Comment:
protocol 2 is not cross-platform compatible: Python 2 protocol 2 is not compatible with Python 3 protocol 2.

Reply:
I'll revert to protocol=0.